### DISCUSSION TITLE
Can Battery Saver / Energy Saver mode be manually set?

### DISCUSSION THREAD

**Discussion by stamminator** [dahall/Vanara#476] [Q&A]
Sorry if this question is off-base. I'm just dipping my toes into WinRT and know very little so far.

Is this setting capable of being changed by user code?

![image](https://github.com/user-attachments/assets/ac092958-c874-4a37-8b6f-33ebd60f0f75)

I believe this is the setting: https://learn.microsoft.com/en-us/uwp/api/windows.system.power.powermanager.energysaverstatus?view=winrt-22621#windows-system-power-powermanager-energysaverstatus

**Comment by dahall** [comment]
My guess is yes, but I couldn't point you to the exact code or method.
